src/CalcManager/UnitConverter.cpp

```diff
@@ -922,6 +923,24 @@ void UnitConverter::TrimString(wstring& returnString)
     }
 }
 
+/// <summary>
+/// Get number of significant digits (integer part + fractional part) of a number</summary>
+/// <param name="value">the number</param>
+unsigned int UnitConverter::GetNumberSignificantDigits(std::wstring value)
+{
+    TrimString(value);
+    unsigned int numberSignificantDigits = static_cast<unsigned int>(value.size());
```

Comment:
What's wrong with `size_t`?  :)

Reply:
The function returns a unsigned int, so a cast is necessary.

Reply:
The function returns an unsigned int, so a cast is necessary.

Reply:
I was mostly poking for fun :)  Although, if the return value is derived from a type with a higher precision there's no imperative to convert to a lower precision type.  It would indicate to me that the return type should also be `size_t`.  That said, I'm not sure how many clients there are of this function and maybe you didn't want to go update.